bridge_ui/package-lock.json

```diff
@@ -87,12 +87,12 @@
     },
     "../sdk/js": {
       "name": "@certusone/wormhole-sdk",
-      "version": "0.5.0",
+      "version": "0.6.1",
       "extraneous": true,
       "license": "Apache-2.0",
       "dependencies": {
-        "@certusone/wormhole-sdk-proto-web": "^0.0.1",
-        "@certusone/wormhole-sdk-wasm": "file:../js-wasm",
+        "@certusone/wormhole-sdk-proto-web": "^0.0.3",
+        "@certusone/wormhole-sdk-wasm": "^0.0.1",
```

Comment:
Just out of curiosity, what is wormhole-sdk-wasm?

Reply:
https://www.npmjs.com/package/@certusone/wormhole-sdk-wasm the wasm lib for invoking webassembly functions (mostly solana functions)